### DISCUSSION TITLE
SSR with Qwik

### DISCUSSION THREAD

**Discussion by ranwahle** [realworld-apps/realworld#976] [WIP Implementations]
Hello, I'm working on SSR with Qwik.

Here is my repository https://github.com/ranwahle/realworld-qwik-app

**Comment by geromegrignon** [comment]
Hi @ranwahle, I can't wait to see it completed: I'm pretty curious about Qwik.

**Reply by ranwahle** [reply]
Thanks. 
You may join me, there are still some pages to implement, such as the login page, and the settings.

**Reply by geromegrignon** [reply]
@guilhermegregio here is an opportunity to team up
